### DISCUSSION TITLE
Is there a way to define a model for an oracle's table that uses a dblink, ex: mytable@myotherserver

### DISCUSSION THREAD

**Discussion by kingkongquelconque** [sequelize/sequelize#16209] [Help & Questions]
Hi,

I want to include a model for querying (only) a table that is on another server.
This table is defined on my Oracle's database via a dblink
ex : 
`SELECT * FROM mySchema.myTable@myOtherServer `
While it works fine in Sql Developer, I can't find a way to query it with sequelize 

Here's an example of the model 
```
const myTableModel = sequelize.define('myTable', {
        field1 : {
            type: DataTypes.INTEGER,
            primaryKey: true 
        },
        field2 : {
            type: DataTypes.STRING 
        }
    }, {
        schema: "mySchema",
        tableName: "myTable@myOtherServer"
    }) ; 
}) ; 
myTableModel.findAll() ; 
```
The reason that it doesn't work... is that sequelize translate the query into something like this :
`SELECT "field1", "field2", FROM "mySchema"."myTable@myOtherServer"  "myTable"`

Where the query sould be (see the quotations near the @)  :
`SELECT "field1", "field2", FROM "mySchema"."myTable"@"myOtherServer"  "myTable"`

I've tried tricking the tableName with something like : 
`tableName: "myTable\"@\"myOtherServer"`
but it gets corrected back to "myTable@myOtherServer" at runtime 

Any ideas ?

**Comment by WikiRik** [comment]
@hjamil-24 maybe you can help out with this as you worked on Oracle integration in sequelize

**Comment by ephys** [comment]
As far as I know this is not something we support, but this sounds very similar to the [`database` feature request](https://github.com/sequelize/sequelize/issues/12449) for postgres and mssql. We'll need to research whether the same option can be used or if we need a `databaseLink` option

**Comment by hjamil-24** [comment]
`dblink` feature is currently not supported on Oracle integration in sequelize as mentioned above. In my opinion, `databaseLink` option might be viable solution and clear to Oracle Database users. Nevertheless, I will take a look on other  better ways in which we can add the support.
